specs/altair/beacon-chain.md

```diff
@@ -309,10 +320,17 @@ def get_base_reward_per_increment(state: BeaconState) -> Gwei:
 
 #### `get_base_reward`
 
-*Note*: The function `get_base_reward` is modified with the removal of `BASE_REWARDS_PER_EPOCH`.
+*Note*: The function `get_base_reward` is modified with the removal of `BASE_REWARDS_PER_EPOCH` and the use of increment based accounting.
 
 ```python
 def get_base_reward(state: BeaconState, index: ValidatorIndex) -> Gwei:
+    """
+    Return the base reward for the validator defined by ``index`` with respect to the current ``state``.
+
+    Note: A validator can optimally earn one base reward per epoch over a long time horizon.
```

Comment:
```suggestion
    Note: An optimally performing validator can earn one base reward per epoch over a long time horizon.
```
(not sure if optimally was used in the right way here, that sentence felt wrong to me)

Reply:
an optimal validator can under optimal conditions

Ha. I wanted to say that it wasn't entirely in their hands. But they need to optimally perform and also have optimal conditions